tests/test_annotate_anndata.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import pandas as pd
+import pytest
+from anndata import AnnData
+
+import genomic_features as gf
+
+genes = gf.ensembl.annotation("Hsapiens", 108).genes()
+
+# Make dummy anndata object
+adata = AnnData(
+    X=np.array([[1, 2, 3], [4, 5, 6]]),
+    obs=pd.DataFrame(index=["cell1", "cell2"]),
+    var=pd.DataFrame(index=["ENSG00000278862", "ENSG00000212694", "ENSG00000113721"]),
+)
+
+
+def test_output():
+    adata_var = gf.annotate_anndata(adata.var, genes)
+    assert isinstance(adata_var, pd.DataFrame)
+
+
+def test_size():
+    annotated_adata_var = gf.annotate_anndata(adata.var, genes)
+    assert annotated_adata_var.shape[0] == adata.n_vars
+
+
+def test_var_names():
+    annotated_adata_var = gf.annotate_anndata(adata.var, genes)
+    assert "var_names" not in annotated_adata_var
+    assert annotated_adata_var.index.equals(adata.var.index)
+    assert annotated_adata_var.index.name == adata.var.index.name
```

Comment:
Could these all operate on the same `annotated_adata_var`?

This could be 

```suggestion
@pytest.fixture(scope="module")
def annotated_adata_var():
    return gf.annotate_anndata(adata.var, genes)

def test_output(annotated_adata_var):
    assert isinstance(annotated_adata_var, pd.DataFrame)


def test_size(annotated_adata_var):
    assert annotated_adata_var.shape[0] == adata.n_vars


def test_var_names(annotated_adata_var):
    assert "var_names" not in annotated_adata_var
    assert annotated_adata_var.index.equals(adata.var.index)
    assert annotated_adata_var.index.name == adata.var.index.name
```